README.md

```diff
@@ -150,16 +150,24 @@ Default options can be overridden using the `removePropTypes` option. These opti
 
 For example, if you are using this plugin in a deployable app, you might want to use the remove mode for your production build (and disable this transform entirely in development for optimal build speeds).
 
-## Classes loose mode
+## Selective loose modes
 
-By default, this preset will compile classes in normal mode. This is safer, but comes with a bundle size and runtime overhead. To [compile classes in loose mode](https://babeljs.io/docs/en/babel-plugin-transform-classes#loose), set the `looseClasses` option to `true`:
+By default, this preset will compile everything in normal mode. This is safer, but comes with bundle size and runtime overhead. We have some options to selectively opt in to loose mode for some features. These options are:
```

Comment:
Maybe we should add some kind of warning, to urge users to be very careful to understand the caveats before enabling loose mode, and that if there's any doubt, to avoid it?